core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -935,7 +947,14 @@ async def confirm_coinjoin(
 
 
 def show_coinjoin() -> None:
-    log.error(__name__, "show_coinjoin not implemented")
+    _RustLayout(
+        trezorui2.show_info(
+            title="CoinJoin in progress.",
+            description="Do not disconnect your Trezor.",
+            button="",
+            time_ms=1,
```

Comment:
`time_ms != 0` means the layout will not draw buttons. I wanted to avoid adding parameter for this but with `time_ms` it's not obvious ... re-doing this.

Reply:
See 50dd2cb12e9c263585b6b7698a41c0fa2fe73abd